lua/autorun/player_extension.lua

```diff
@@ -155,4 +155,103 @@ function PLAYER:UsingWeapon(cls)
     local c = self:GetActiveWeapon()
 
     return IsValid(c) and c:GetClass() == cls
+end
```

Comment:
Can you just put all this in chatcommands/stuck.lua please?

Reply:
yeah one sec